src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -514,6 +514,7 @@ private function createTypeNotFoundMessage(TypedReference $reference, string $la
 
             if ($r->isInterface() && !$alternatives) {
                 $message .= ' Did you create a class that implements this interface?';
```

Comment:
```suggestion
                $message .= ' Did you create an instantiable class that implements this interface?';
```

Reply:
@nicolas-grekas Thanks for the comment. But can we be sure that this is the best way?
The purpose of my PR is to provide the user with as much information about the error as possible. This is especially true for beginners.
I would also like to receive comments from other team members, what they think about this.

Reply:
A private constructor is not the only reason: an abstract class is another one that could end up with this message. I'm all for clearer messages, but here this looks like documentation in a message, and it's not complete. The adjective "instantiable" is enough this to hint the brain on the correct path IMHO?

Reply:
Hm, maybe you're right. Thanks for the tip